sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -38,6 +38,15 @@ static const struct snd_soc_acpi_adr_device rt711_0_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device rt711_2_adr[] = {
+	{
+		.adr = 0x000220025D071100ull,
```

Comment:
can you verify the rt711 address? This should be the SDCA version and there should be a 3 in there. that points to the older version